src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -351,6 +360,9 @@ public abstract partial class DbDataReader : System.Collections.IEnumerable, Sys
         public abstract bool Read();
         public System.Threading.Tasks.Task<bool> ReadAsync() { return default(System.Threading.Tasks.Task<bool>); }
         public virtual System.Threading.Tasks.Task<bool> ReadAsync(System.Threading.CancellationToken cancellationToken) { return default(System.Threading.Tasks.Task<bool>); }
+        System.Data.IDataReader System.Data.IDataRecord.GetData(int ordinal) { return default(System.Data.IDataReader); }
+        public void Close() { }
+        public DataTable GetSchemaTable() { return default(DataTable); }
```

Comment:
The way it is right now, if I have an instance of `SqlDataReader` or some other `DbDataReader` subclass, and I look at it with intellisense, `GetSchemaTable` is going to show up in the list and will be callable because it is public.

```c#
SqlDataReader reader = ...;
DataTable table = reader.GetSchemaTable(); // works
```

But I'm assuming the `GetSchemaTable` method isn't actually useful in CoreFx and is only there because it's on the interface.

If instead `GetSchemaTable` was implemented explicitly, the following wouldn't work (which, I believe, is what you want):

```c#
SqlDataReader reader = ...;
DataTable table = reader.GetSchemaTable(); // doesn't work, doesn't show up in intellisense
```

The only way to call `GetSchemaTable` would be to work with the interface itself, e.g. by casting the reader to the interface.

```c#
SqlDataReader reader = ...;
IDataReader readerInterface = reader;
DataTable table = readerInterface.GetSchemaTable(); // works
```